src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-detection-step.html

```diff
@@ -0,0 +1,83 @@
+<!-- PROGRESS BAR -->
+<div class="col-md-8"
+     data-ng-if="$ctrl.detectionStepProgression < 100">
+    <span data-translate="tools_lineDiagnostics_detection_step_test"></span>
+    <oui-progress>
+        <oui-progress-bar data-type="info"
+                          data-value="$ctrl.detectionStepProgression"></oui-progress-bar>
+    </oui-progress>
+</div>
+
+<div class="col-md-4"
+     data-ng-if="$ctrl.currentLineDiagnostic.status === 'waitingRobot'">
+    <p>
+        <oui-spinner size="s"></oui-spinner>
+        <span data-translate="tools_lineDiagnostics_waitingRobot"></span>
+        <span data-translate="tools_lineDiagnostics_please_wait"></span>
+    </p>
+</div>
+
+<div class="col-md-8"
+     data-ng-if="$ctrl.detectionStepProgression === 100 &&
+                 $ctrl.currentLineDiagnostic">
+    <div data-ng-if="$ctrl.currentLineDiagnostic.hasQuestionToAnswer('modemIsSynchronized')">
+        <form class="form-group"
+              novalidate name="firstStepQuestionForm">
+            <oui-field data-label="{{:: 'tools_lineDiagnostics_detection_step_modem_question' | translate }}"
+                       data-label-popover="{{:: 'tools_lineDiagnostics_detection_step_modem_question_help' | translate }}">
+                <oui-radio-toggle-group class="d-inline-block"
+                                        data-model="$ctrl.currentLineDiagnostic.data.answers.modemIsSynchronized"
+                                        data-on-change="$ctrl.startPoller()"
+                                        data-ng-show="!$ctrl.loading">
```

Comment:
`ngIf` ?

Reply:
`oui-radio-toggle-group` is inside a `oui-field` component.
`ng-if` would delete `oui-radio-toggle-group` so the `oui-field` will make an error because no element is inside when ng-if is true